.github/workflows/static-checks.yml

```diff
@@ -0,0 +1,73 @@
+name: Static checks - misspell and shellcheck
+on: push
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: egde-resource-mgr-'${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  static-checks:
+    runs-on: ubuntu-latest
+    env:
+      SUMMARY_FILE: summary.log
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      # Need to run this 1st, so that the other log files do not cause unnecessary findings
+      # tox already runs pylint
+      - name: Run misspell
+        if: always()
+        run: |
+            # Remove the vendor folder - we don't need to check that.
+            rm -rf vendor
+            curl -L -o ./install-misspell.sh https://git.io/misspell
```

Comment:
There's a different repo we should pull this from. A more maintained version is in:  https://github.com/golangci/misspell